doc/user/content/sql/update.md

```diff
@@ -10,14 +10,18 @@ menu:
 
 ## Syntax
 
-{{< diagram "update-stmt.svg" >}}
+```mzsql
```

Comment:
This page and the VALUES page ... Initially, I thought I could do some minor content updates as I went but realized after this page that I should scope just to the syntax since some pages require more than minor content updates.